Setup/UpgradeSchema.php

```diff
@@ -64,24 +64,59 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
                 $oldColumnName
             );
         }
-        $columnTaxConfig = 'valitor_price_includes_tax';
+        /**
+         * Replace database priceInclusive column
+         */
+        $columnTaxConfig    = 'altapay_price_includes_tax';
+        $oldColumnTaxConfig = 'valitor_price_includes_tax';
         if (!$setup->getConnection()->tableColumnExists($setup->getTable($orderTable), $columnTaxConfig)) {
-            $setup->getConnection()->addColumn(
+            if ($setup->getConnection()->tableColumnExists($setup->getTable($orderTable), $oldColumnTaxConfig)) {
+                $setup->getConnection()->changeColumn(
+                    $setup->getTable('sales_order'),
+                    $oldColumnTaxConfig,
+                    $columnTaxConfig,
+                    [
+                        'type'     => Table::TYPE_BOOLEAN,
+                        'nullable' => true,
+                        'visible'  => false,
+                        'comment'  => 'Whether catalog prices entered from Magento Admin include tax.',
+                    ]
+                );
+            } else {
+                $setup->getConnection()->addColumn(
+                    $setup->getTable($orderTable),
+                    $columnTaxConfig,
+                    [
+                        'type'     => Table::TYPE_BOOLEAN,
+                        'nullable' => true,
+                        'visible'  => false,
+                        'comment'  => 'Whether catalog prices entered from Magento Admin include tax.',
+                    ]
+                );
+            }
+        } elseif ($setup->getConnection()->tableColumnExists($setup->getTable($orderTable), $oldColumnTaxConfig)) {
+            $setup->getConnection()->dropColumn(
                 $setup->getTable($orderTable),
-                $columnTaxConfig,
-                [
-                    'type'     => Table::TYPE_BOOLEAN,
-                    'nullable' => true,
-                    'visible'  => false,
-                    'comment'  => 'Whether catalog prices entered from Magento Admin include tax.',
-                ]
+                $oldColumnTaxConfig
             );
         }
-
-        $valitorTableToken = $setup->getTable('valitor_token');
-        if ($setup->getConnection()->isTableExists($valitorTableToken) != true) {
-            $valitorTableToken = $setup->getConnection()->newTable(
-                $valitorTableToken
+        /**
+         * Replace database table from sdm_valitor to sdm_altapay
+         */
+        if ($setup->getConnection()->isTableExists($setup->getTable('sdm_valitor')) == true
+            && $setup->getConnection()->isTableExists($setup->getTable('sdm_altapay')) == false
+        ) {
+            $setup->getConnection()->renameTable($setup->getTable('sdm_valitor'), $setup->getTable('sdm_altapay'));
```

Comment:
Why not using TransactionInterface::TABLE_NAME instead of 'sdm_altapay' ?

Reply:
Updated accordingly.